assets/src/components/v2/dup/rotation_index.tsx

```diff
@@ -0,0 +1 @@
+export const ROTATION_INDEX = 1;
```

Comment:
for dup v1, the rotation index was set in the file to 0, and the readme indicates we're still working with digits 0-2. Should this be 0?

Reply:
The committed index is not the index that is used in the packaging. In the `for` loop part executed to create the packaged zips, we export the index needed for each loop and that index is saved to this file. This file is mostly there because so that way we don't have to worry about it after the packaging is finished. One thing that might be useful is committing index 2 to the file. That will always be the last index saved so we would never need to worry about a diff when we run the script.

Reply:
> This file is mostly there because so that way we don't have to worry about it after the packaging is finished.

Could you explain this differently? Not quite following. But cool, glad it's not a necessary value 👍

Reply:
A better solution might actually be just adding this file to `.gitignore`. If the file doesn't exist, the script will create it so no dev needs to worry about it existing before they can package the app.

Reply:
Thanks for the explain! Why do we need to save rotation_index in a file at all? Is it used later?

Reply:
The main reason is because the code for the packaged app needs access to the variable. After compilation, the file is no longer necessary. To help with that, I added it to `.gitignore` in [this PR](https://github.com/mbta/screens/pull/1719). With it ignored, we can just forget it ever existed and let the script worry about it.